SR-1.5 updates (#402)
lidofinance/docs: docs/guides/oracle-spec/validator-exit-bus.md

[Validators Exit Bus](/contracts/validators-exit-bus-oracle) is an oracle that ejects Lido validators when the protocol requires additional funds to process user withdrawals.

There are two stages of selecting validators for exit: Covering Demand in WQ and Boosted Exits

A report calculation consists of 6 key steps:

1. Calculate withdrawals amount to cover with ether.
2. Calculate ether rewards prediction per epoch.
3. Calculate withdrawal epoch for next validator eligible for exit to cover withdrawal requests if needed.
4. Prepare validators exit order queue to fulfill withdrawals.

> 5. Extend with forced to exit validators up to report limit or until there is no force requests.

Reviewer: ```suggestion
5. Extend exit list with forced to exit validators (`targetLimitMode` is set to the boosted mode) up to the report limit or until there are no forced requests.
```

Note that th described algorithm is looking for a validator to exit only among those that can be exited, while using the projected number of validators, which includes non-existent yet validators. It's only weights, so there is no misconception here.

The final exit order predicate sequence to fulfill withdrawal requests:

1. Validator whose operator with the lowest number of delayed validators

> 2. Validator whose operator with the highest number of forced targeted validators to exit

Reviewer 2: ```suggestion
2. Validator whose operator with the highest number of boosted targeted validators to exit
```

- validators count that are not yet in CL
- validators that are in CL and are not yet requested to exit and not on exit
- validators that are in CL and requested to exit but not on exit and not requested to exit recently 

> - target type (forced/soft) and target validators count

Reviewer 2: ```suggestion
- target type (soft/boosted) and target validators count
```
